tee-worker/ts-tests/identity.test.ts

```diff
@@ -353,7 +353,7 @@ describeLitentry('Test Identity', 0, (context) => {
             context.substrateWallet.alice,
             alice_txs,
             'identityManagement',
-            ['IdentityVerified']
+            ['IdentityVerified', 'VerifyIdentityFailed'] // allow possible failed events to be collected
```

Comment:
Shouldn't we collect other possible events in other cases, for example `RemoveIdentityFailed` in case of removing identity ?

Another way is to wait for example 10 blocks and strictly check what events were emitted and do not pass if there is something more than expected.

Reply:
In fact,we don't want to accommodate unexpected events in the expected process; We will throw unexpected events. In fact, it will eventually timeout because we expect to receive three `IdentityVerified` events, but in reality, we only receive two. So, [events.length === txsLength ](https://github.com/litentry/litentry-parachain/blob/dev/tee-worker/ts-tests/common/transactions.ts#L163)cannot be true. I suggest closing this pr,merge [this pr](https://github.com/litentry/litentry-parachain/pull/1624) first.The issue with @Kailai-Wang has been optimized in the pr,we will throw an unexpected event.

Reply:
I think current implementation is fine, as in it will only try to get the IdentityVerified event, before timeout (which will fail test).
In this case, the current dev branch can already work with the VerifyIdentityFailed event. Evidence as below:

<img width="1304" alt="image" src="https://github.com/litentry/litentry-parachain/assets/120463031/31721cd4-accb-4363-b6e1-158090055c4f">